terraform/roles.tf

```diff
@@ -288,3 +288,58 @@ resource aws_iam_role_policy_attachment ec2_readonly {
   role       = aws_iam_role.caascad_operator.name
   policy_arn = "arn:aws:iam::aws:policy/AmazonEC2ReadOnlyAccess"
 }
+
+data aws_iam_policy_document caascad_monitoring_cloud_services_assumerole_policy {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "AWS"
+      identifiers = [var.caascad_operator_trusted_arn]
```

Comment:
Can I use this variable that is also used for `caascad-operator` ?

Reply:
@mvienne-orange : please reuse the same assumerole_policy than one of operator